assets/app/view/game/borrow_train.rb

```diff
@@ -78,9 +78,10 @@ def remaining_trains
 
         rows = @depot.upcoming.group_by(&:name).flat_map do |_, trains|
           names_to_prices = trains.first.names_to_prices
+          train_count = trains.first.unlimited ? '∞' : trains.size
           [h(:div, names_to_prices.keys.join(', ')),
            h(:div, names_to_prices.values.map { |p| @game.format_currency(p) }.join(', ')),
-           h(:div, trains.size)]
+           h(:div, train_count)]
```

Comment:
If the count is only used here, do we need the var?

Reply:
0a1c592fb